### PR TITLE
style: use camelCase functions and misc formatting

### PR DESCRIPTION
**Pull request by aakropotkin**
- style: use camelCase function names.
- style: misc separator and newline additions.

### CODE REVIEW

File: include/flox/core/exceptions.hh [hunk 1]
```diff
@@ -64,45 +69,56 @@ class FloxException : public std::exception
...
+   * virtual methods.
+   * - We don't want to force overriding what() in every child class, because
+   *   that would be a pain.
+   * - We can't do that when calling what(), because what() returns a char *
+   *   and is const, we don't have anywhere to store that dynamic information.
+   * - We can't do it at construction time, because we can't call virtual
+   *   methods.
+   */
   [[nodiscard]] std::string
   what_string() const noexcept;
```
Comment (mkenigs) [inline include/flox/core/exceptions.hh:117]: Do you want this camelCase?

**Review by mkenigs — APPROVED** [review]

**Comment by mkenigs** [comment]
Wonder if it's worth using clang-tidy? https://clang.llvm.org/extra/clang-tidy/checks/readability/identifier-naming.html#readability-identifier-naming

File: include/flox/core/exceptions.hh (hunk 1 above)
Comment (aakropotkin) [reply include/flox/core/exceptions.hh:117]: Oh yeah good catch

**Comment by zmitchell** [comment]
Are there `clang-format` rules that can be tweaked to catch these things?

**Comment by mkenigs** [comment]
> Are there `clang-format` rules that can be tweaked to catch these things?

No but there are `clang-tidy` rules like the one I linked

**Comment by mkenigs** [comment]
But I will try to follow conventions and feel free to point out when I'm not in review @aakropotkin

**Pull request merged**